src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,9 @@
+$logo-size: 40px;
+$color-lightblue: #00acdc;
```

Comment:
There are two variables ($color-lightblue and $blue-accent-color) that hold the same color. It's better to have only one variable for a single color to minimize redundancy and potential confusion.